try_obsconfig_model.ipynb

```diff
@@ -0,0 +1,614 @@
+{
+ "cells": [
```

Comment:
Remove or make this a proper `docs/tutorial`?

Reply:
this was committed accidentally (as were the commits to EventSource) I'll remove both.